day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,45 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(*topping)
+    topping.each do |t|
+      self.toppings << t
+    end
+    self.toppings.sort
+    # self.toppings << topping
+
+  end
+
+  def remove_topping(*topping)
```

Comment:
The way this is setup, we're removing multiple toppings. What would be a better name in that case?

Alternatively, how could you simplify it to remove just one topping?